cognee/modules/retrieval/lexical_retriever.py

```
@@ -0,0 +1,121 @@
+                return
+
+            logger.info("Initializing LexicalRetriever (shared) from graph engine")
+            graph_engine = await get_graph_engine()
+            nodes, _ = await graph_engine.get_graph_data()
+
+            chunk_count = 0
+            for node in nodes:
+                try:
+                    chunk_id, document = node
+                except Exception:
+                    continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)